src/jit/codegenlinear.cpp

```diff
@@ -1947,3 +1947,63 @@ void CodeGen::genCodeForCast(GenTreeOp* tree)
     }
     // The per-case functions call genProduceReg()
 }
+
+#if !defined(_TARGET_64BIT_)
+//-----------------------------------------------------------------------------
+//
+// Code Generation for Long integers
```

Comment:
Do we really need such a header?

Reply:
No; that was copied from the ARM version. I'll delete it.